test/test_distributed.py

```diff
@@ -778,6 +778,17 @@ def forward(self, x):
                 x = self.fc3(x)
                 return F.softmax(x, dim=1)
 
+        def model_step(model):
+            for layer in model.modules():
```

Comment:
Sorry I missed this one. Can you change it to use `.parameters()` too? There's nothign special about the weights, it should work for biases too